rpa-manager/modules/ROOT/pages/processautomation-prepare-project-application.adoc

```diff
@@ -75,7 +75,7 @@ For an explanation of the properties, see  <<form-create-process-applications, *
 
 == Deactivate or Activate a Process Application
 
-Deactivate an application if it is temporarily unavailable and you don't know when it will be available again.
+Deactivate an application if it is temporarily unavailable and you don't know when it is available again.
```

Comment:
This is another case where I would use the future tense.

Reply:
Vale suggested this edit :)